setup/linter/README.md

```diff
@@ -0,0 +1,22 @@
+# io.js Build Linter Setup
+
+These machines (currently just one) are run as part of the jenkins job,
+linting any code before passing it on to compile/test.
+
+To set up hosts, make sure you add them to your ssh config first:
+```
+Host iojs-linter
+  HostName 104.236.229.96
+```
+
+Note that these hostnames are also used in the ansible-inventory file. The IP addresses will need to be updated each time the servers are reprovisioned.
+
+To set up a host, run:
+
+```text
+$ ansible-playbook -i ../ansible-inventory ansible-playbook.yaml
+```
+
+**Users**: The ansible-vars.yaml file contains a list of users who's GitHub public keys are pulled and placed into
```

Comment:
Good catch. I copied this from another vm, looks like most configs has the same typo. I'll remedy them in a new PR.